### ISSUE TITLE
Offer comfortable bug reporting from the browser 

### ISSUE THREAD

**Issue by aeickhoff** [citybound/citybound#292]
- [ ] Create an issue directly from the Citybound client (what about GitHub registration?)
- [ ] Collect system info
- [ ] Automatically find similar already reported bugs (by stack trace or description)

**Comment by Flambe** [comment]
Is it possible to have a bot post the issue to github?

**Comment by aeickhoff** [comment]
You mean in case the reporter doesn't want to create an account?

**Comment by Flambe** [comment]
Yeah exactly. I feel like it'd be easier/more seamless for the player if they aren't redirected to github to login. Might be a bit off putting if they just want to leave a little feedback and suddenly the need to sign up for a random website.

**Comment by aeickhoff** [comment]
I understand and I was thinking the same, on the other hand registering gives me the opportunity to reach back to the exact person reporting it and makes them get notifications about progress regarding their issue.

**Comment by Flambe** [comment]
Ah yeah I didn't think of that. Maybe they could have an option to sign up/log in if they want to receive updates/are happy to give more details?

**Comment by MiniDigger** [comment]
I feel like for the current situation where these are pretty much development builds it's ok to require an GitHub account. This can be changed later once a broader audience plays the game, but I think currently it's just fine.
